src/main/java/org/mitre/synthea/export/CPCDSExporter.java

```diff
@@ -259,28 +254,28 @@ private String patient(Person person, long time) throws IOException {
    * @param careplan    The careplan itself
    * @throws IOException if any IO error occurs
    */
-  private void coverage(String personID, String encounterID, CarePlan careplan,
-      String payerId, String type, UUID groupId, String groupName) throws IOException {
-
+  private void coverage(String personID, Long start, Long stop, String payerId, String type,
```

Comment:
Please use the primitive `long` for consistency with other code, unless there is a particular reason for using the object form `Long`.